grow/pods/routes.py

```diff
@@ -186,6 +186,7 @@ def match_error(self, path, status=404):
 
     def reset_cache(self, rebuild=True, inject=False):
         if rebuild:
+            self.pod.podcache.reset()
             self._build_routing_map(inject=False)
```

Comment:
Just a note - I can't remember all the specifics around the routing map cache but that is something we could probably move to the new podcache as well eventually.